cli/build.gradle.kts

```diff
@@ -31,3 +35,38 @@ if (isMac) tasks.withType<JavaExec>().configureEach {
 }
 
 tasks.withType<JavaCompile>().configureEach { CommonTasks.picocli(this) }
+
+val genDir = file("build/native-config")
+val configFile = file("reflect-config.txt")
+
+val generateReflectionConfig = tasks.register<org.aya.gradle.GenerateReflectionConfigTask>("generateReflectionConfig") {
+  outputDir = genDir
+  inputFile = configFile
+}
+
+graalvmNative {
+  binaries {
+    named("main") {
+      imageName.set("aya")
+      mainClass.set("org.aya.cli.Main")
+      verbose.set(true)
+      fallback.set(false)
+      debug.set(System.getenv("CI") == null)
+      sharedLibrary.set(false)
+      configurationFileDirectories.from(genDir)
+      useFatJar.set(true)
+    }
+  }
+
+  binaries.configureEach {
+    javaLauncher.set(javaToolchains.launcherFor {
+      languageVersion.set(JavaLanguageVersion.of(17))
+      vendor.set(JvmVendorSpec.matching("GraalVM Community"))
+    })
+  }
```

Comment:
@ice1000

Reply:
If I don't need to run the ni task, do I still have to install graalvm?